cmd/kpod/ps.go

```diff
@@ -144,18 +146,19 @@ func psCmd(c *cli.Context) error {
 	if err := validateFlags(c, psFlags); err != nil {
 		return err
 	}
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "could not get config")
+
+	// latest, and last are mutually exclusive.
+	if c.Int("last") >= 0 && c.Bool("latest") {
+		return errors.Errorf("last and latest are mutually exclusive")
```

Comment:
If I'm just seeing this error for the first time while being new to kpod, will I know what to do with it?  How do I fix "latest and last are mutually exclusive"?

Reply:
Shouldn't `--all` be mutually exclusive as well?

Reply:
I dont believe so, I think that is an error.  I think you should be able to get the last or latest regardless of its running status, which is what all determines.

Reply:
fyi, my code follows docker behavior.

Reply:
right, the current code prints out the latest or last n regardless of the container status. I just remember @rhatdan telling me to make them all mutually exclusive, but if this is what docker does sure :)

Reply:
They should all be mutual exclusive, don't understand when you would ever use them in combination.

Reply:
all can be used with latest or last.  latest and last *are* mutually exclusive.  This follows the exact way docker does it.

Reply:
Where docker does not make sense we can fix.

What does --all --latest mean, Do I see all or just lates?
What does --latest --all mean?

Just block --latest --all and -last

Reply:
--all doesn't mean show all containers - just show containers that are not running in addition to containers that are running. So it doesn't conflict with --last or --latest.